desktop/plugins/network/ManageMockResponsePanel.tsx

```diff
@@ -217,14 +221,21 @@ export function ManageMockResponsePanel(props: Props) {
               props.responses,
             );
           }}>
-          <Glyph
-            name="plus-circle"
-            size={16}
-            variant="outline"
-            color={colors.blackAlpha30}
-          />
+            <Glyph
```

Comment:
Looks like prettier wasn't run on this PR. VSCode (and most other IDEs have a plugin for it). If you have trouble running it, let us know, we can run it on our side as well.

Reply:
Is "prettier" the same as "yarn fix" (which I did forget to run)?